package.json

```diff
@@ -27,6 +27,8 @@
     "doc:typedoc": "typedoc --out doc/typedoc/ src/",
     "generate": "npm-run-all clean compile test:unit doc",
     "build": "npm-run-all clean compile",
+    "watch": "npm-run-all test watch:compile",
```

Comment:
Running test here is useless, mainly because it's only run once, but running tests and dev-code together is not something you would normally do; you either run your tests, or testing your implementation.